client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/bootstrap/BootstrapManager.java

```diff
@@ -16,26 +16,26 @@
 
 package org.kaaproject.kaa.client.bootstrap;
 
-import java.util.List;
-
```

Comment:
There are a lot of files with changed formatting which are unrelated to this task. Please revert formatting for such files in order to have changes only related to this task.